lib/inspec/resources/platform.rb

```diff
@@ -55,46 +69,49 @@ def in_family?(family)
       @platform.family_hierarchy.include?(family)
     end
 
+    # rubocop:disable Style/HashSyntax
     def params
-      # rubocop:disable Layout/AlignHash
-      h = {
-        name:     name,
+      {
+        name: name,
         families: families,
-        release:  release,
-        arch:     arch,
-      }
-      # rubocop:enable Layout/AlignHash
+        release: release,
+        arch: arch,
+      }.tap { |h| h.delete(:arch) if in_family?("api") }
+    end
 
-      h.delete :arch if in_family?("api") # not applicable if api
+    def supported?(supports, recursive: false)
+      return true if supports.nil? || supports.empty?
```

Comment:
We might still wanna keep logic to raise this error `raise ArgumentError, supports is nil. unless supports`

Reply:
Happy to put it back in. I think this was more of a switch by vscode rubocop cleaning.

Reply:
Thanks!